tools/restatectl/src/commands/log/mod.rs

```diff
@@ -130,3 +137,24 @@ pub fn deserialize_replicated_log_params(segment: &Segment) -> Option<Replicated
         _ => None,
     }
 }
+
+pub async fn get_logs(connection: &ConnectionInfo) -> anyhow::Result<LogsMetadata> {
+    let req = GetMetadataRequest {
+        kind: MetadataKind::Logs.into(),
+        sync: false,
+    };
+
+    let mut response = connection
+        .try_each(None, |channel| async {
+            let mut client = NodeCtlSvcClient::new(channel)
+                .accept_compressed(CompressionEncoding::Gzip)
+                .send_compressed(CompressionEncoding::Gzip);
+
+            client.get_metadata(req).await
+        })
+        .await?
+        .into_inner();
```

Comment:
Can we make sure to get the latest `Logs` if we can obtain it from multiple nodes?